reactor-core/src/main/java/reactor/core/scheduler/Scheduler.java

```diff
@@ -47,9 +47,28 @@
 	 *
 	 * @return the {@link Disposable} instance that let's one cancel this particular task.
 	 * If the {@link Scheduler} has been shut down, throw a {@link RejectedExecutionException}.
+	 *
```

Comment:
nitpick: remove the newlines between `@` javadoc tags (keep one between last line of body and first tag)

Reply:
(+ a few other occurrences in the javadocs below)